docs/scanning-for-extension-points.md

```diff
@@ -0,0 +1,40 @@
+# Scanning for Extension Points
+
+On initialization, essentially as soon as a FakeItEasy type is
+accessed, FakeItEasy uses reflection to look for internal and
+user-supplied extension points. In most cases, there is no _need_ for
+users to define any extensions, but they may be used to enhance the
+power and usability of FakeItEasy.
+
+There are currently three kinds of extension points defined:
+
+* [Custom Dummy Creation](custom-dummy-creation.md) rules,
+* Fake Configurators

```

Comment:
Actually, there never was! Hard to believe, I know, but we didn't have one.
In fact, I had only the most tenuous of ideas of what a "Fake Configurator" was until, well, pretty near the time we started on 2.0 work, I think.
I'm lazy and figured that everyone had gotten along okay without such a page up 'til now, so I didn't bother making one for the conversion.
If you'd like, I can write something up.